content/en/tracing/setup/istio.md

```diff
@@ -14,29 +14,27 @@ further_reading:
 - link: "https://github.com/DataDog/dd-opentracing-cpp"
   tag: "Source Code"
   text: "Datadog OpenTracing C++ Client"
-aliases:
-  - /tracing/servicemesh/istio
 ---
 
-Support for Datadog APM is available for Istio on Kubernetes clusters.
-This was first available in Istio v1.1.3.
+Datadog APM is available for Istio v1.1.3+ on Kubernetes clusters.
 
 ## Configuration
 
 ### Datadog Agent Installation
 
-The Datadog Agent must be [installed][k8s-install] and [APM enabled][apm-enabled].
-The `hostPort` setting needs to be uncommented for Istio sidecars to connect to the agent and submit traces.
+1. [Install the Agent][1]
+2. [Make sure APM is enabled for your Agent][2].
+3. Set the `apm_non_local_traffic` parameter to `true` and uncomment the `hostPort` setting so that Istio sidecars can connect to the Agent and submit traces.
```

Comment:
This gets automatically enabled when `KUBERNETES=yes` and `DD_APM_ENABLED=true`.
It's not an additional step a user needs to do.

Reply:
Updated :) does the new wording works for you?

Reply:
Yup!